docs/zk2kraft.html

```diff
@@ -155,44 +155,49 @@ <h5 class="anchor-heading">Removal metrics</h5>
+                <li><code>kafka.controller:type=KafkaController,name=FencedBrokerCount</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=UncleanLeaderElectionsPerSec</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=IdleRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=ControllerChangeRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=LeaderElectionRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=TopicChangeRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=TopicDeletionRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=PartitionReassignmentRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=AutoLeaderBalanceRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=ManualLeaderBalanceRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=ControlledShutdownRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=IsrChangeRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=LeaderAndIsrResponseReceivedRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=LogDirChangeRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=ControllerShutdownRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=UncleanLeaderElectionEnableRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=TopicUncleanLeaderElectionEnableRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=ListPartitionReassignmentRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=UpdateMetadataResponseReceivedRateAndTimeMs</code></li>
+                <li><code>kafka.controller:type=ControllerStats,name=UpdateFeaturesRateAndTimeMs</code></li>
                 <li><code>kafka.controller:type=KafkaController,name=MigratingZkBrokerCount</code></li>
                 <li><code>kafka.controller:type=KafkaController,name=ZkMigrationState</code></li>
-            </ul>
-            <p>
-                Kafka remove all zookeeper dependencies, so the metrics is removed.
-            </p>
-        </li>
-        <li>
-            <p>
-                Remove the metrics which is monitoring requests waiting in the topic purgatory
-            </p>
-            <ul>
                 <li><code>kafka.server:type=DelayedOperationPurgatory,name=PurgatorySize,delayedOperation=topic</code></li>
```

Comment:
Could you please align the order of attributes? `type`,`delayedOperation`,`name` WDYT?